dasharo-compatibility/cpu-cores-count.robot

```diff
@@ -58,6 +58,7 @@ CCC003.001 Check core count (HT Enabled, P: All, E: 0) (Ubuntu)
     Reset To Defaults Tianocore
     Save Changes And Reset
     Set UEFI Option    ActiveECores    0
+    Power Cycle On
```

Comment:
(applies to all changes in this file)

Why is this needed? `Set UEFI Option` ends with `Save Changes And Reset`, is it broken?

Reply:
Info: resolved by: https://github.com/Dasharo/open-source-firmware-validation/commit/836169f6940ec10d65c81c4c3a302f8c06905068